cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -170,23 +152,37 @@
         },
         {
             "events": [
+                {
+                    "created_on": "2018-07-06T12:30:08.123456789Z",
+                    "msg": {
+                        "channel": {
+                            "name": "Nexmo",
+                            "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
+                        },
+                        "text": "Ryan Lewis",
+                        "urn": "tel:+12065551212",
+                        "uuid": "9bf91c2b-ce58-4cef-aacc-281e03f69ab5"
+                    },
+                    "step_uuid": "c34b6c7d-fa06-4563-92a3-d648ab64bccb",
+                    "type": "msg_received"
```

Comment:
previously such events were considered caller events and not included in the session event log - so callers do have to be aware `msg_received`, `run_expired`, `wait_timed_out`, `contact_changed` and `environment_changed` are not events that they have to act on